homeassistant/components/meteoalarm/sensor.py

```diff
@@ -0,0 +1,99 @@
+"""Sensor for MeteoAlarm.eu."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_COUNTRY = 'country'
+CONF_PROVINCE = 'province'
+CONF_LANGUAGE = 'language'
+
+ATTRIBUTION = ("Information provided by MeteoAlarm, time delays between this"
+               "website and the www.meteoalarm.eu website are possible, for"
+               "the most up to date information about alert levels as"
+               "published by the participating National Meteorological"
+               "Services please use www.meteoalarm.eu")
```

Comment:
Only the attribution please. Additional details should go into the documentation.

Reply:
I cleaned it up